### ISSUE TITLE
Feature request: Elapsed time since last log item column

### ISSUE THREAD

**Issue by mriffey** [CobaltFusion/DebugViewPP#339]
Recently I was working with another dev on some performance issues. At one point, i took the dbv++ output, pasted it into a blank email (to get the spacing formatted), then took that and pasted it into Excel. I then added a column for Elapsed time (this step) and added a formula to calculate for each line how much time had elapsed since the last log entry. A simple process, but kind of cumbersome.

Adding an elapsed time since last log entry column would be useful for such testing. Example screenshot below. 

![image](https://user-images.githubusercontent.com/2301643/45182493-9e361280-b1de-11e8-9a38-13f94ddec260.png)

**Comment by janwilmans** [comment]
A you aware of then clock icon at then top of the screen ? It switches from system clock to steady clock. Steady clock displays elapsed time from the top of the view and the view can be reset to a certain line you want.

**Comment by janwilmans** [comment]
Also:
![image](https://user-images.githubusercontent.com/5933444/45208814-d21b4180-b28b-11e8-8a31-2e06d2bb30dc.png)

**Comment by mriffey** [comment]
Yep. Im talking about an additional column next to that which shows the time since the last log entry, not the current delta time since the log started. The screenshot I included shows that (created in excel) and the column to the left of the elapsed column in the screenshot is the delta time from a debugview log.

**Comment by janwilmans** [comment]
Ah, right. sounds like that would not be too difficult to add, any suggestion how to fit it in the UI ? (menu-wise) ?

**Comment by mriffey** [comment]
Column-wise, Id put it next to the time column, regardless of format. Menu-wise, Id put it on View with the other time-related items.

**Comment by janwilmans** [comment]
I have thought about this and I really like the idea

**Comment by mriffey** [comment]
Great news, Jan. Ill be happy to test it when it’s available.

 

Mark
